### PR TITLE
Round token supply number in token list

### PR DESCRIPTION
**Pull request by buberdds**
Token supply has too many decimal points

### CODE REVIEW

File: .changelog/702.trivial.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Round token supply number in token list
```
[the remark this answers is not in the document]
Comment (csillag) [reply .changelog/702.trivial.md]: I see it's solved now.

File: src/app/components/Tokens/TokenList.tsx [hunk 2]
```diff
@@ -129,7 +130,7 @@ export const TokenList = (props: TokensProps) => {
           align: TableCellAlign.Right,
         },
         {
-          content: token.total_supply,
+          content: <RoundedBalance value={token.total_supply} ticker={token.symbol} />,
```
Comment (csillag) [inline src/app/components/Tokens/TokenList.tsx:133]: Actually I prefer this way of formatting the supply: https://github.com/oasisprotocol/explorer/blob/master/src/locales/en/translation.json#L215

With rounded balance:
image


Via i18next: 
image

Comment (csillag) [reply src/app/components/Tokens/TokenList.tsx:133]: But I also understand that it is not great with decimals.
I think we need a way that satisfies both criteria.

Can we have the same grouping in RoundedBalance?

(Also, maybe we could consider to resurrect #333?)

Comment (buberdds) [reply src/app/components/Tokens/TokenList.tsx:133]: Ok, I think we are good here. As discussed during meeting, RoundedBalance logic will be adjusted in another PR.

**Pull request merged**